src/bin/qml/pages/RightPanel.qml

```diff
@@ -84,7 +84,7 @@ Item {
     function openFriends(userId, screenName, account, pushMode) {
         var args = {"user_id": userId}
         var params = {
-            title: qsTr("%1's friends").arg(screenName),
+            title: qsTr("%1's follower").arg(screenName),
```

Comment:
Damn, it's not that, thanks for the heads-up:
It should be something like: "%1 is following"